Core/Tpp.cs

```diff
@@ -47,7 +58,12 @@ private async Task OnMessage(Message message)
                     .Process(commandName, parts.Skip(1).ToImmutableList(), message);
                 if (result.Response != null)
                 {
-                    _logger.LogDebug($"> {message.User}: {result.Response}");
+                    if (message.MessageSource == MessageSource.Chat)
+                        await _chat.SendMessage($"@{message.User.TwitchDisplayName} {result.Response}");
+                    else if (message.MessageSource == MessageSource.Whisper)
+                        await _chat.SendWhisper(message.User, result.Response);
```

Comment:
for most commands this will work fine. but there will inevitably be commands that want to controle where they post their response. I haven't figured out a way to add this to the command processor that wouldn't needlessly bloat it and instead thought the commands that _do_ need that will just have to depend on the chat and send messages themselves. Does that sound about right?

Reply:
might be a good idea to address that when/if we come across that, since it might need to be specific to the command how that's implemented